sympy/printing/pycode.py

```diff
@@ -186,6 +188,27 @@ def _expand_reduce_binary_op(self, op, args):
                 self._expand_reduce_binary_op(args[Nhalf:]),
             )
 
+    def _print_Add(self, expr):
+        # An alternative could be to use `math.fsum`, but that would limit arguments
+        # to floats essentially, and arguably a user interested in maximizing precision
+        # is already better served by the mpmath printer. And finally, subclassing and
+        # overriding this remains an option.
+        if len(expr.args) >= self._settings['num_terms_sum']:
+            return 'sum([%s], start=%s)' % (
```

Comment:
this should use `module_format` and `builtins.sum` like here:

https://github.com/sympy/sympy/blob/677fc6b88491f265fd30ee8d9b818b0f9fb861e1/sympy/printing/pycode.py#L271-L280

(e.g. clashing import names/definitions)

Actually, now that I look at it, that code looks bugged since it doesn't use module_format :)